lib/PhpParser/Node/ContainsStmts.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpParser\Node;
+
+interface ContainsStmts extends \PhpParser\Node {
```

Comment:
I think this needs `@property` tag

```suggestion
/**
 * @property Stmt[]|null $stmts
 */
interface ContainsStmts extends \PhpParser\Node {
```

`null` still needed tho for property definition, as `$stmts` property itself can be null.